pages/studentPortal/quizcomponent/career-results.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Button } from "../../../components/ui/Button";
+import QuizNavbar from "./quiznavbar";
+
+const Career_Results = () => {
+  return (
+    <div className="grid  w-full mx-auto text-center justify-items-center">
+      <QuizNavbar></QuizNavbar>
+      <div className="font-bold mt-4 text-2xl ">YOUR RESULTS</div>
+      <div className=" text-lg font-semibolds ">
+        Here are jobs in tech you’re compatible with. Click on a career to learn
+        more!
+      </div>
+      <img
+        src="/images/software-engineer.png"
+        alt="Software Engineer"
+        className=" p-4 rounded-[40px]  md:h-60 md:w-80 "
```

Comment:
i'm ok with the hardcoded 40px but did you try using `object-fit` as well in addition to `rounded-lg`? If this works, im fine with this.